src/ActionModal/components/UndelegationModal.vue

```diff
@@ -105,7 +77,6 @@
           class="tm-field-addon"
           placeholder="0"
           type="number"
-          :is-disabled="session.addressRole === `controller`"
```

Comment:
@mariopino why did you remove this? Controller accounts shouldn't be able to change the staked amount right?

Reply:
Only controllers can reduce the staked amount

Reply:
OK, I was wrong, the can and it is stash accounts who cannot do anything unstake related it seems. Just checked